internal/speakerlist/speakerlist.go

```diff
@@ -234,6 +236,18 @@ func (sl *SpeakerList) members() map[string]struct{} {
 	return members
 }
 
+// updateJoinList: mlJoinList = (k8sSpeakers - mlSpeakers)
+// must be called with SpeakerList mutex locked
```

Comment:
This function exists to avoid copy pasting 2 time the same code.
It is called when we either update k8sSpeakers or mlSpeakers, and when we do those updates we are already holding the lock. As golang doesn't have reentrant locks, a comment on a private function is the way to go IMO